### PR TITLE
Create User- Backend Functionality 

### PR DESCRIPTION
**Pull request by jbleggi**
Type of change

  - [X] New feature
  - [ ] Bug Fix

Check the correct boxes

  - [X] This code broke nothing
  - [ ] This code broke some stuff
  - [ ] This code broke everything

Implements/Fixes:

  - Created User backend functionality
  - Route, Controller, Model created
  - Validations helper created to verify correct user entry
  - Some edge cases/sad paths tested (missing or invalid input)
  - RSpec tests for Controller & Route are passing and at 100% SimpleCov 

Testing Changes:

  - [X] I have fully tested my code 
  - [X] All tests are passing
  - [ ] Some tests are failing
  - [ ] All tests are failing

  - [X] No previous tests have been changed
  - [ ] Some previous tests have been changed
  - [ ] All of the previous tests have been changed (Please describe what in the world happened that all of the previous tests needed changing.)

Checklist:

  - [X] I have reviewed my code
  - [ ] The code will run locally
  - [ ] My code has no unused/commented out code
  - [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: app/validators/user_validator.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# app/validators/user_validator.rb
```
Comment (elysableu) [inline app/validators/user_validator.rb:1]: So glad this helped!

**Review by elysableu — APPROVED** [review]
This looks great overall and the tests look very thorough!
